Use Cases/implement-binance-future-pnl-analysis-page.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Implement Binance Future PNL Analysis page"
```

Comment:
Should we mention Phoenix Live View in the title?

Reply:
Yes, I updated